SUMMARY.md

```diff
@@ -0,0 +1,28 @@
+## Table of Contents
+
+* [Ternary Operator](content/TernaryOperator/TernaryOperator.q.md) ** - minutes**
+    > Ternary operator is a shorthand way of doing if/else statements. It holds three operands: a condition, a result for true, and a result for false. You will learn how you can use the ternary operator in PHP.
+
+* [Type Hinting](content/TypeHinting/TypeHinting.q.md) ** - minutes**
+    > Discover how you can use type hinting to specify the expected data type in a function declaration.
+
+* [Cookies](content/Cookies/Cookies.q.md) ** - minutes**
+    > Learn how to create/retrieve cookies, modify cookie values, delete cookies, and check if cookies are enabled.
+ 
+* [Sessions](content/Sessions/Sessions.q.md) ** - minutes**
+    > Learn how to start a PHP session, get PHP session variable values, modify a PHP session variable, and destroy a PHP session.
+
+* [Serialization](content/Serialization/Serialization.q.md) ** - minutes**
+    > Serialization in PHP is used for storing and passing values while retaining their type and structure. In this lesson, you will learn how you can use `serialize()` and `unserialize()` methods.
+
+* [Error Handling](content/ErrorHandling/ErrorHandling.q.md) ** - minutes**
+    > Error handling refers to the process of catching errors detected in the program that requires proper action. Find out how you can handle errors using different types of error handling, error logs, and error handling functions.
+    
+* [Catching Exceptions](content/CatchingExceptions/CatchingExceptions.q.md) ** - minutes**
+    > Catching an exception is the new object-oriented way of dealing with errors through exception handling. It searches for a `catch` statement that can handle the type of exception. You will learn how you can handle errors with `try-catch` statements and basic exception classes.
+
+* [Throwing Exceptions](content/ThrowingExceptions/ThrowingExceptions.q.md) ** - minutes**
+    > Throwing an exception requires an exception object with a parameter of the `Exception` constructor as the error message and then throw it. Discover how you can throw an exception using the `throw new` statement.
+
+* [Custom Exceptions](content/CustomExceptions/CustomExceptions.q.md) ** - minutes**
+    > Discover how you can create your own exception handler using custom class and functions that inherit the properties of the PHP exception class.
```

Comment:
What particular `custom class and functions` that we can use to create our own exception handler? Based on my experience, a `custom exception` is a user defined error handling function within a program that will handle the errors when called. The only difference is that you are the one who created that function to handle the error. I think we need to rephrased this statement. Thanks.